lib/graphql/subscriptions/anycable_subscriptions.rb

```diff
@@ -243,6 +265,33 @@ def fetch_channel_istate(channel)
       def redis_key(prefix)
         "#{config.redis_prefix}-#{prefix}"
       end
+
+      def executor_class_job
+        custom_class = config.async_broadcasting["class"]
+
+        return Adapters::BaseJob unless custom_class
+
+        Object.const_get(config.async_broadcasting["class"])
+      end
+
+      def perform(event, object)
```

Comment:
Where do we use this method?

Reply:
It was pre-alfa version of the code, sorry :)